recipes/trata/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "trata" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LLNL/trata/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e94cc7a302c203c1654d987fc9a8e453cf45a95e659366911e4dfde84b1ace83
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  noarch: python
```

Comment:
```suggestion
  noarch: python
  entry_points:
    - trata = trata.trata:__init__
```

Reply:
@doutriaux1 and @olson59 it was the lack of entrypoints that caused  the Windows failures. It should be fine now.

Reply:
Thanks @ocefpaf !